setup/sql/6_hotfixes.php

```diff
@@ -1712,3 +1712,8 @@
     ));
 }
 ?>
+<#74>
+<?php
+$ilDB->manipulateF('DELETE FROM settings WHERE keyword = %s', ['text'], ['enable_block_moving']);
+$ilDB->manipulateF('DELETE FROM il_block_setting WHERE type LIKE %s', ['text'], ['pd%']);
```

Comment:
@iszmais Please use `ilDbInterface::like` and be aware of using multile `%%`, which is required in some cases.